src/libaktualizr/storage/embed_schemas.py

```diff
@@ -54,6 +57,18 @@ def apend_migration(migration_path, header_file):
             header_file.write("\",\n")
         apend_migration(os.path.join(migration_folder, migration_list[-1]), header_file)
         header_file.write("\"\n};\n")
+
+        header_file.write("extern const std::map<int, std::string> {}_schema_rollback_migrations = {{".format(prefix))
```

Comment:
We could get away with a `std::vector` where the missing elements are empty strings.
For consistency with the other migration list, also I'm wary of what a static std::map compiles to.